stack/rest/src/main/java/org/apache/usergrid/rest/applications/users/UserResource.java

```diff
@@ -164,6 +167,81 @@ public JSONWithPadding setUserPasswordPut( @Context UriInfo ui, Map<String, Obje
         return new JSONWithPadding( response, callback );
     }
 
+    @GET
+    @RequireSystemAccess
+    @Path("credentials")
+    public JSONWithPadding getUserCredentials(@QueryParam("callback") @DefaultValue("callback") String callback )
+            throws Exception {
+
+        logger.info( "UserResource.setUserPassword" );
+
+
+        final ApiResponse response = createApiResponse();
+        response.setAction( "get user password" );
```

Comment:
maybe let's be more explicit on what this returns as it's not the password in plaintext. 'get user password hash' or something?